services/hermes-rl-trajectory/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM python:3.13-slim
+
+WORKDIR /app
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends curl && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY services/hermes-rl-trajectory /app
+ENV PYTHONPATH=/app/src
+RUN pip install --no-cache-dir --upgrade pip && \
```

Hadolint (2.14.0)

[error] Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead.

(SC2261)